### DISCUSSION TITLE
Unable to scan QR code

### DISCUSSION THREAD

**Discussion by PainxLuna** [Authenticator-Extension/Authenticator#1230] [Q&A]
It was fine, but it stopped reading QR codes, I try to read but nothing appears, sometimes it just says, the QR code could not be scanned :c

**Comment by mymindstorm** [comment]
The libraries we use to scan QR codes aren't very reliable, the best thing to do is be careful to get only the QR code when scanning and trying a few times. Otherwise, try manual entry.
